build-setup/clion/using_clion.md

```diff
@@ -39,7 +39,7 @@ By default `CLion` will opt for generating a `Ninja` build system. While this is
 better compatibility with building from the command line it is preferred to configure `CLion` to generate
 `Unix Makefiles`. To do this either select `Unix Makefiles` from the `Generator` drop-down or type 
 `-G "Unix Makefiles"` in the`CMake options` field. Add  e.g. `-j 10` to the `Build options` field, select your
-desired build type (either`Release` or `Debug`) and we're done. Make sure to leave the `Build directory` field
+desired build type (e.g.`Release` or `Debug`) and we're done. Make sure to leave the `Build directory` field
```

Comment:
I think we should recommend people work day-to-day with `RelWithDebInfo` and only switch to something different if absolutely necessary for some specific debugging exercise or investigation.  We've now got it set up so that `RelWithDebInfo` gives our desired blend of performance and debuggability, and it's what we ship to our users, so it's best that people develop with that day-to-day where possible.

Also, should we recommend _against_ using `Default` for the build type? Does selecting `Default` actually do a build that's identical to `RelWithDebInfo`? If it does then maybe `Default` is OK (although it does create a separate CMake build directory). But if it doesn't then it seems to be actively confusing.

Reply:
> Also, should we recommend _against_ using `Default` for the build type?

A bit of experimentation indicates that the `Default` build type uses flags that are equivalent to those corresponding to `RelWithDebInfo` (at least on macOS), but I think it might be worth stating that it's best to explicitly choose one of the other build types, to avoid any confusion.

Reply:
> to avoid any confusion

It also has the advantage that builds from the command line can reuse object files already built by CLion builds, and vice-versa.  If CLion is using `Default` and the command line is using `RelWithDebInfo` then they have separate object files built with the same flags.